frontend/admin/src/js/components/pool/CreatePool.tsx

```diff
@@ -188,185 +171,82 @@ export const CreatePoolForm: React.FunctionComponent<CreatePoolFormProps> = ({
               name="key"
```

Comment:
I'm unable to test this because the "key" input returns the following error:

> Expected type KeyString at value.key; The given value "TEST_KEY" did not match the regex /^[a-z]+[_a-z0-9]*$/ 

Should we add some instruction for users on the format since it is a required field?

Reply:
key field will be removed in #3561 so I don't think you should worry too much about that one
I've just put strings of random letters for the time being like "asdasdasfas"

Reply:
hang on, I should double check if it being removed or changed

Reply:
Sounds good, everything else working well. Once you figure this out let me know and I should be able to approve 👍

Reply:
Key Field will be gone gone per https://github.com/GCTC-NTGC/gc-digital-talent/issues/3561#issuecomment-1204232963 

Another big thing, currently there is unsureness as to whether classifications, job titles or both will used in Create/Edit Pool, so for the time being I mirrored the Edit Pool with select Classification bit but left the job title stuff in for use later
Mirroring Edit Pool for the time was a decision made in Huddle